custom-recipes/nlp-visualization-wordcloud/recipe.py

```diff
@@ -28,6 +28,8 @@
     remove_stopwords=params["remove_stopwords"],
     remove_punctuation=params["remove_punctuation"],
     case_insensitive=params["case_insensitive"],
+    max_words=params["max_words"],
+    color_list=params["color_list"],
```

Comment:
Same, this change is already in `main` so it shouldn't appear in this branch. Need to merge main into this branch.